pkg/cluster/kubeclient/ready.go

```diff
@@ -158,11 +145,11 @@ func (u *kubeclient) MasterIsReady(computerName ComputerName) (bool, error) {
 	return ready.PodIsReady(u.client.CoreV1().Pods("kube-system"), "controllers-"+computerName.toKubernetes())()
 }
 
-func (u *kubeclient) nodeWaitForReady(ctx context.Context, computerName ComputerName) error {
+func (u *kubeclient) WaitForReadyWorker(ctx context.Context, computerName ComputerName) error {
```

Comment:
After going back to this for a while, it's not obvious to me why a wait for readiness method sets the node to unschedulable. Since this is exported now, probably a good chance to add godoc.

Reply:
It seems the node is set to schedulable (unschedulable=false). Still unclear why this is needed for worker nodes.

Reply:
I don't believe it is, I intend to remove it.

Reply:
(in a subsequent PR)